go.mod

```diff
@@ -23,3 +22,7 @@ require (
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
 )
+
+replace github.com/gphotosuploader/google-photos-api-client-go => github.com/pdecat/google-photos-api-client-go v0.0.0-20190728103106-52b4c499bcab
```

Comment:
Why are you using this replace? We can merge your code into `github.com/gphotosuploader/google-photos-api-client-go`

Reply:
This replace is needed to make it build here until the google-photos-api-client-go PR is merged.